### PR TITLE
remove duplicate sending msg to full channel

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:
This block is confusing
When we write data failed, we don't need to write again, IMOH, we can remote the second write operation.
Before https://github.com/kubeedge/kubeedge/pull/2664 , we use `select` to try to write data to channel again, but I think it's not needed.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by vincentgoat — APPROVED** [review]
LGTM

File: staging/src/github.com/kubeedge/beehive/pkg/core/channel/context_channel.go [hunk 1]
```diff
@@ -173,7 +173,6 @@ func (ctx *Context) SendToGroup(moduleType string, message model.Message) {
 		case ch <- message:
 		default:
 			klog.Warningf("The module %s message channel is full, message: %+v", module, message)
-			ch <- message
```
Comment (fisherxu) [inline staging/src/github.com/kubeedge/beehive/pkg/core/channel/context_channel.go:176]: If the channel is full, should we wait to write or discard? If just discard, the message maybe lost? If wait to write, whether the order of the message can be ensured between multi goroutine?

Comment (gy95) [reply staging/src/github.com/kubeedge/beehive/pkg/core/channel/context_channel.go:176]: yes, i think we can just drop the messages when the message channel is full.

Comment (fisherxu) [reply staging/src/github.com/kubeedge/beehive/pkg/core/channel/context_channel.go:176]: Need to check the design model or where use the sendToGroup, whether we should discard :)

**Pull request closed without merging**